### ISSUE TITLE
do mdroid support moodle 3?

### ISSUE THREAD

**Issue by Vinodh134** [praveendath92/MDroid#79]

**Comment by praveendath92** [comment]
No. Not at the moment. This is because we are currently using deprecated functions to support older Moodle versions instead. This is on the road map though.
